cmd/tailscale/cli/switch.go

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cli
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/peterbourgon/ff/v3/ffcli"
+	"tailscale.com/ipn"
+)
+
+var switchCmd = &ffcli.Command{
+	Name:       "switch",
+	ShortHelp:  "Switches to a different Tailscale profile",
+	LongHelp:   `"tailscale switch" switches between logged in profiles`,
+	ShortUsage: "switch [<name>] [--list]",
```

Comment:
It's more like `"switch --list\nswitch <name>\n"` (you can do multiple lines, right?)